docs/testing-guidelines/TestRoadmap.md

```diff
@@ -79,7 +79,6 @@ We need to be sure that we can easily enable remoting for the non-Windows platfo
 * Our current multi-machine tests do not test the connection code, they simply execute test code remotely and retrieve results and assume a good connection.
 The infrastructure used for these tests is STEX which is not an open environment.
 We will need to create automation to create and configure the test systems in the test matrix and then invoke tests on them.
-It is not clear that our current CI systems can accommodate our needs here as neither AppVeyor or Travis can supply us with all of the OS images needed.
```

Comment:
This is still true of Azure DevOps as we have not solved windows 7 and some other Operating Systems.

Reply:
Fixed